conda.recipe/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+    name: mindsdb
+    version: "2.10.2"
+
+source:
+    git_url: https://github.com/mindsdb/mindsdb.git
+    git_rev: "v2.10.2"
+
+extra:
+    channels:
+        - conda-forge
+    deploy:
+        - anaconda-client
+
+build:
+    noarch: python
+    entry-points:
+        - mindsb = mindsdb.__main__:main
+
+    script:
+        python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    host:
+        - python
+        - pip
+    build:
+        - yaml
+    run:
+        - python>=3.5
+
+dependencies:
+        - pip:
+            - mindsdb_native == 2.13.0
```

Comment:
Thank you. @Darknez07 can this infor be pulled from the requirements.txt so we don't need to manage two separate ones?
@George3d6 thoughts?

Reply:
yes it can be we can use jinja

Reply:
even versions can be pulled